### ISSUE TITLE
EnergyPlus workflow failing and output suffix

### ISSUE THREAD

**Issue by JasonGlazer** [NREL/EP-Launch#59]
On windows I'm getting:
```
Exception in thread Thread-11:
Traceback (most recent call last):
  File "C:\Users\Jason\AppData\Local\Programs\Python\Python36\lib\threading.py", line 916, in _bootstrap_inner
    self.run()
  File "I:\epRepo\EP-Launch\eplaunch\interface\workflow_processing.py", line 44, in run
    workflow_response = self.workflow_instance.main(self.run_directory, self.file_name, self.workflow_main_args)
  File "C:\EnergyPlusV8-9-0\workflows\energyplus.py", line 327, in main
    success, errors, warnings, runtime = EPlusRunManager.get_end_summary(end_file_path)
  File "C:\EnergyPlusV8-9-0\workflows\energyplus.py", line 21, in get_end_summary
    contents = open(end_file_path, 'r').read()
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\EnergyPlusV8-9-0\\test\\5\\5Zone_Unitary_HXAssistedCoilout.end'

```

Which is probably related to the output suffix of 'C' being used which is making a file called: 5Zone_Transformer.end without the "out" before the ".end" extension. Is that how it is working on Linux and MacOS? We need to settle on one of these output formats or else fail more gracefully.

**Comment by Myoldmopar** [comment]
I have a branch going with a few fixes in it, and I'm working on this one now.  I have already added a try block around the call to the workflow's "main" function, so that it does end more gracefully.  I tested it by hacking a workflow file with the purposely wrong "end" filename:
![image](https://user-images.githubusercontent.com/849824/44937176-9f2df680-ad3d-11e8-9fd4-a14d30347b01.png)

Note:
 - the popup window stays open with a little informative message
 - nothing comes out on stderr
 - the status bar also has a message

A question still remains about the end file naming itself.  I've never seen this problem.  It's worked fine for me wherever I tested it, but I don't recall exactly where I've tested it.  If we need a trap for different platforms inside the E+ workflow because E+ behaves differently on them with respect to output file names, then so be it, we just need to modify the workflows.
